index.html

```diff
@@ -182,6 +182,25 @@ <h2> About <br> Myself</h2>
             </ul>
           </div>
           <hr>
+
+          <section class="section-form">
+            <div class="form-div">
+             <h3>Contact me</h3>
+             <p>If you have an application you are 
+               interested in developing,
+               a feature that you need built or a project that 
+               needs coding. I’d love to help with it </p>
```

Comment:
- [ ] You have done a great job implementing the requested changes 👏 👏 . Though I still suggest that you stick to the Figma design provided including the padding. Ensure that the `p` padding is greater than the form inputs to make it slightly small in terms of width relative to the form inputs ☺️ 

- Your design 😎 
![text](https://user-images.githubusercontent.com/99394374/214549078-ca90f842-28c5-4596-9eed-fc86c80432c7.PNG)

- Figma design 😎 
![text fig](https://user-images.githubusercontent.com/99394374/214548897-5de91595-7982-4008-adff-eaf35875d9b5.PNG)